src/compiler/compile/nodes/shared/Expression.ts

```diff
@@ -391,23 +390,16 @@ export default class Expression {
 
 		if (declarations.length > 0) {
 			block.maintain_context = true;
-			declarations.forEach(declaration => {
-				block.chunks.init.push(declaration);
-			});
+			block.chunks.init.push(...declarations);
```

Comment:
👍🏻 

but, do you know there may be a maximum number of arguments for a function call? (Ref: https://stackoverflow.com/a/22747272/1513547)

so calling a function with spread may not always be the best option